pwiz_tools/Skyline/Model/Databinding/Entities/ColumnToolTips.resx

```diff
@@ -294,6 +294,9 @@ values.</value>
     <value>Declustering Potential for the precursor ion according to
 instrument/vendor specific default declustering potential equation within Skyline</value>
   </data>
+  <data name="DeconvolutedPrecursorArea">
+    <value>The observed MS1 area adjusted based on the predicted isotope distribution of the precursors</value>

```

Comment:
Explain more clearly. e.g. Does it only impact overlapping isotope distributions like deuterated Leucine, or does it impact even a single precursor? What is it doing? A regression of some sort, yes? Is it a regression on the log values? or the linear values? I assume the isotope distribution is the independent value and the measured are the dependent value? And then you use the resulting equation measured-area = isotope-proportion * b1 + b0?

Reply:
Good point. I'll try to come up with a better sentence.

If we are fitting only one isotope distribution, then it is a least squares linear regression through the origin. The X coordinates are the predicted distribution values (normalized so that the total distribution sums to 1), and the Y coordinates are the observed peak areas. The fitted slope is the number that gets reported as the DeconvolutedPrecursorArea.
There is no y-intercept term. This happens to be appropriate because the Y coordinates were the background subtracted areas.

When fitting multiple overlapping isotope distributions, it's a least squares fit. It's described in the 1966 paper Least Squares Analysis and Simplification of Multi-Isotope Mass Spectra J. I. Brauman.

Reply:
Okay, I get it. Y = mx1 + mx2 + mx3 where x1 + x2 + x3 = 1 means that Y = m. But, yeah, I think you need to mention "linear regression through the origin" in the description. I think most readers would assume "linear regression" implies "least squares". Maybe even reference "(Brauman, Anal. Chem. 1966)" for overlapping isotope distributions. That gets me there with a Google search.